docs/reference/commandline/build.md

```diff
@@ -96,22 +96,22 @@ For example, run this command to use a directory called `docker` in the branch
 `container`:
 
 ```bash
-$ docker build https://github.com/docker/rootfs.git#container:docker
+$ docker build https://github.com/docker/rootfs#container:docker
```

Comment:
Since the server supports git urls as a remote, I wonder if the client should detect this and perform the API request with the git url instead of creating the context locally.